paperclip-storage-ftp.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency("rake")
   gem.add_development_dependency("daemon_controller", ">= 1.1.0")
   gem.add_development_dependency("activerecord")
+  gem.add_development_dependency("byebug") unless RUBY_PLATFORM =~ /java/
```

Comment:
ok interesting, didn't know about that until now. the related commit is [reverted](https://github.com/xing/paperclip-storage-ftp/commit/e586814113a1b9eec16217919e7bfa896dfc6e4d).